src/vsr/replica.zig

```diff
@@ -757,7 +761,6 @@ pub fn ReplicaType(
                 } else {
                     // Even if op_head_certain() returns false, a DVC always has a certain head op.
```

Comment:
I think you can remove this comment; the above "Guaranteed since our durable..." comment refers to the same thing.

Reply:
Mhmmm, isn't this comment different? I always thought it meant that we don't need to call `op_head_certain` for the case where `log_view < view` because we rely _only_ on DVC (without looking at the journal) to infer head in this case. 🤔

Reply:
Actually, let me just update that in the comment for clarity!

Reply:
Ah right, thank you!